### DISCUSSION TITLE
Not getting 7.4.0 image in Portainer?

### DISCUSSION THREAD

**Discussion by drjjr2** [zwave-js/zwave-js#2594] [Q&A] [answered]
Anyone know what would keep me from being able to do the usual to get the latest image using Portainer? Seems like I'm stuck on 7.3.0. Maybe I've accidentally changed something with the container?

**Comment by drjjr2** [comment, marked as answer]
Could be because I'm an idiot and not paying attention to the node-zwave-js vs zwavejs2mqtt container...mistakes were made.

**Reply by robertsLando** [reply]
@drjjr2 z2m 4.1.0 is out now with zwave-js 7.4.0

**Reply by drjjr2** [reply]
Thank you. Didn't get my coffee early enough today I guess.
